base/task.jl

```diff
@@ -728,7 +728,7 @@ Print an error log to `stderr` if task `t` fails.
 
 # Examples
 ```julia-repl
-julia> Base._wait(errormonitor(Threads.@spawn error("task failed")))
+julia> wait(errormonitor(Threads.@spawn error("task failed")))
```

Comment:
`_wait` doesn't throw as compared to `wait` which is why it was written like this I think. However, https://github.com/JuliaLang/julia/pull/53685 exposed the non-throwing behavior with the `throw` keyword:
```suggestion
julia> wait(errormonitor(Threads.@spawn error("task failed")); throw = false)
```

Reply:
Ah nice, added :+1: